packages/pyodide-kernel/src/kernel.ts

```diff
@@ -198,6 +198,9 @@ export class PyodideKernel extends BaseKernel implements IKernel {
         );
         break;
       }
+      case 'execute_return': {
```

Comment:
Wondering if such message should be named something else, or prefixed with an underscore or similar? Otherwise it looks like it may be an official message part of the Jupyter protocol, while it isn't.

Reply:
sure, `jpk:` would be solid. It's not really _private_, and a downstream may well want to react to it. We'd pay a minimal tax to do the `.split` but adding more fields at this time seems even worse and more breaking.

Most of these should be hanging off a `const` or `namespace` somewhere (but not `enum`: those can break in round-tripped JSON settings).

Reply:
So, should we rename the message to `jpk:execute_return` after all??